pkg/runtime/framework/plugins/mpi/mpi_test.go

```diff
@@ -807,6 +807,72 @@ func TestValidate(t *testing.T) {
 				).
 				Obj(),
 		},
+		"runtime does not have Launcher but TrainJob has only 1 numNodes": {
+			info: runtime.NewInfo(
+				runtime.WithMLPolicySource(utiltesting.MakeMLPolicyWrapper().
+					WithMLPolicySource(*utiltesting.MakeMLPolicySourceWrapper().
+						MPIPolicy(ptr.To[int32](1), ptr.To(trainer.MPIImplementationOpenMPI), nil, ptr.To(true)).
+						Obj(),
+					).
+					Obj(),
+				),
+				runtime.WithPodSet(constants.Node, ptr.To(constants.AncestorTrainer), 1, corev1.PodSpec{}, corev1ac.PodSpec().
```

Comment:
```suggestion
				runtime.WithPodSet(constants.Node, nil, 1, corev1.PodSpec{}, corev1ac.PodSpec().
```

Actually, MPI node does not have any ancestor.